package.json

```diff
@@ -2,8 +2,10 @@
   "name": "libp2p-noise",
   "version": "1.1.2",
   "main": "dist/src/index.js",
+  "types": "dist/src/dist/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "src"
```

Comment:
why do you include the src ? its just taking space in node_modules, a consumer should only need the js source files and .d.ts.

Reply:
for source maps links. Its common for typescript projects to provide source files for people that debug code.